nucypher/cli/deploy.py

```diff
@@ -262,16 +263,8 @@ def deploy(click_config,
             for tx_name, txhash in transactions.items():
 
                 # Wait for inclusion in the blockchain
-                try:
-                    receipt = deployer.blockchain.wait_for_receipt(txhash=txhash)
-                except TimeExhausted:
-                    raise  # TODO: Option to wait longer or retry
-
-                # Examine Receipt # TODO: This currently cannot receive failed transactions
-                if receipt['status'] == 1:
-                    click.secho("OK", fg='green', nl=False, bold=True)
-                else:
-                    click.secho("Failed", fg='red', nl=False, bold=True)
+                receipt = deployer.blockchain.client.w3.eth.waitForTransactionReceipt(txhash)
```

Comment:
```suggestion
                receipt = deployer.blockchain.w3.eth.waitForTransactionReceipt(txhash)
```

Reply:
Is there a reason that `w3` is exposed both in `blockchain` and `blockchain.client`?

Reply:
> Is there a reason that w3 is exposed both in blockchain and blockchain.client?

It's a good question, perhaps @vepkenez can help clarify, this has to do with how we are providing compatibility across clients, wrapping thier `w3` instances.